examples/resources/couchbase-capella_cluster_onoff_schedule/resource.tf

```
@@ -0,0 +1,14 @@
+resource "couchbase-capella_cluster_onoff_schedule" "new_cluster_onoff_schedule" {
+  organization_id = "organization_id"
+  project_id      = "project_id"
+  cluster_id      = "cluster_id"
+  timezone        = "timezone(ex:US/Hawaii)"
+  days = [
+    for day in var.days : {
+      state = "state"
+      day   = "day(ex:Monday)"
```

@SaicharanCB just add the actual values-
```suggestion
      day   = "Monday"
      from =  {
                  hour   = 12
                  minute = 30
                }
       to = {
                  hour   = 14
                  minute = 30
                }
```

---

Also @SaicharanCB  can you do similarly in this file- examples/resources/couchbase-capella_free_tier_cluster_on_off/resource.tf
https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/324/files#diff-4eb7052ca462d49130c6fabe2586db2122b92259f7d85768c66f5f79a4844a49R5
add actual value for state like `state           = "off"`